pkgs/os-specific/linux/realmd/default.nix

```
@@ -0,0 +1,80 @@
+{ stdenv
+, fetchFromGitLab
+, openldap
+, libkrb5
+, packagekit
+, polkit
+, libxslt
+, intltool
+, glib
+, pkg-config
+, systemd
+, autoreconfHook
+, samba
+, adcli
+, oddjob
+, sssd
+, bash
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "realmd";
+  version = "0.17.0";
+
+  srcs = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = pname;
+    repo = pname;
+    rev = version;
+    sha256 = "1c6q2a86kk2f1akzc36nh52hfwsmmc0mbp6ayyjxj4zsyk9zx5bf";
```

```suggestion
    hash = "sha256-bpX+0/T6E9ml98rcVQGrVXMHRYHWDPanCk7MaZAS2LA=";
```

sha256 has been deprecated in favor of hash